analyzers/tests/SonarAnalyzer.UnitTest/TestCases/PropertiesAccessCorrectField.NetFramework.vb

```diff
@@ -0,0 +1,34 @@
+﻿Imports System.Windows
+Imports System.Windows.Controls
+Imports System.Windows.Media
+
+Public Class Repro3442
+    Inherits  UserControl
+
+    Public Shared ReadOnly BackgroundColorProperty As DependencyProperty = DependencyProperty.Register("BackgroundColor", GetType(Color), GetType(Color), New PropertyMetadata(""))
+    Public Shared ReadOnly ForegroundColorProperty As DependencyProperty = DependencyProperty.Register("ForegroundColor", GetType(Color), GetType(Color), New PropertyMetadata(""))
+
+    Private backgroundColor_ As Color
+    Private foregroundColor_ As Color
+
+    Public Property BackgroundColor As Color
+        Get
+            Return GetValue(BackgroundColorProperty)
```

Comment:
could you add a test with GETVALUE, too? it's fine to not support it to keep things simple, but to document that VB is case insensitive